doc/lua_api.txt

```diff
@@ -5380,20 +5380,49 @@ Definition tables
             place = <SimpleSoundSpec>,
             place_failed = <SimpleSoundSpec>,
         },
-        drop = "",
+
+        drop = "", --[[
         ^ Name of dropped node when dug. Default is the node itself.
-        ^ Alternatively:
+          Using a table allows multiple items, random selection among items,
+          palette control and tool filtering:
         drop = {
             max_items = 1,  -- Maximum number of items to drop.
             items = {  -- Choose max_items randomly from this list.
+                -- When using a shovel, they will get a drop about 60% of the time.
+                {
+                    -- Tool names are matched with `==`.
+                    tools = {
+                        "default:shovel_wood", "default:shovel_stone",
+                        "default:shovel_bronze", "default:shovel_steel",
+                        "default:shovel_mese", "default:shovel_diamond",
+                    },
+                    rarity = 2,  -- Probability of dropping is 1 / rarity.
+                    items = { "default:sand" },  -- Items to drop.
+                },
+                {
+                    -- The next entry uses "~", so this list should be its exact
+                    -- opposite.  If this list is not exhaustive, there are extra
```

Comment:
The next entry appears to be lines 5406 to 5413, which is identical to the previous one apart from seeming to show a rarity value change for a different node. The part with ~ doesn't start until 5419 and seems to have a separate comment introducing it. This should probably be structured better, with less repetition and comments more tightly linked to what they reference.